ui-components/src/blocks/DashBlocks.scss

```diff
@@ -24,13 +24,15 @@
   @apply bg-white;
   @apply p-4;
   @apply pt-8;
+  @apply pb-8;
   @apply text-tiny;
 
   @screen md {
     @apply rounded-lg;
     @apply border;
     @apply p-8;
     @apply mb-8;
+    height: 253px;
```

Comment:
In general we want to avoid explicit pixel numbers like this. This number only works for this exact content, but we want the cards to be able to take a flexibly sized input and still have two cards side-by-side be the same height. I imagine we'll want to use flex and flex grow for the container. Below is one example of an issue with this explicit height:
<img width="1048" alt="Screen Shot 2022-04-26 at 12 19 56 AM" src="https://user-images.githubusercontent.com/65367387/165234859-250b8b11-e5f3-4fe6-abb9-b2a724fc8cc4.png">